### ISSUE TITLE
Incorporate RFC2119 styling

### ISSUE THREAD

**Issue by fantasai** [w3c/tr-design#326]
The manual of style has this to say:

https://www.w3.org/Guide/manual-of-style/#RFC
> When these key words are used in the RFC sense, make them UPPERCASE, enclose them in the em element, and style them with CSS class of rfc2119 to make the UPPERCASE readable. [...]
>
> ` .rfc2119 {  text-transform: lowercase;  font-variant: small-caps;  font-style: normal; }`

We should probably pull this styling into base.css

**Comment by fantasai** [comment]
See also https://lists.w3.org/Archives/Public/spec-prod/2023AprJun/0016.html which asks whether this convention should be 

- optional or required
- altered in some way to improve screenreader compatibility

**Comment by fantasai** [comment]
(If we want to make it required, we should probably make sure ReSpec and Bikeshed can auto-generate the requisite markup.)

**Comment by tabatkins** [comment]
Yeah that markup is awkward as all hell to write out manually every single time.  If we want to actually make this required, I'd probably have Bikeshed, like, autotransform `2119may` into the markup, and then flag every normative use of 2119 language that *isn't* marked up.

**Comment by tabatkins** [comment]
But I'm happy to just *not* do it, and remove it from the Manual of Style too. CSS has *never* marked up its 2119 language in this way. (It also uses alternative wording in several places anyway, as noted in our Conformance sections, so if someone is depending on the literal word "MUST" or whatever to stand out and be noticeable, they'll miss most of the normative requirements already.

**Comment by fantasai** [comment]
Previous discussion, fwiw: https://github.com/w3c/tr-design/issues/126
